.travis/run_smoke_tests

```diff
@@ -74,6 +74,7 @@ function startBot() {
     -Dtriplea.name=Bot01Testing \
     -Dtriplea.lobby.host=localhost \
     -Dtriplea.lobby.port=3304 \
+    -Dtriplea.lobby.https.port=8080 \
```

Comment:
Typically HTTPS is 443, whereas 8080 is typically used as alternative port for HTTP (80) which might be confusing.

Anyways what I'm trying to say is that IMO this suggests that the transport (i.e. TLS) is an essential part for the communication to work, so if `https` is just supposed to represent the HTTP based lobby system, I'd actually vote to drop the s.

Reply:
I don't think I'm really thrilled that we call it 'https' or 'http' at all, we're bubbling up technology details.

We do call it 'https' in other places, notably client settings, I think we shot ourselves in the foot already. Of course, until we get some local certs installed and start using https locally, the port must be 8080. The intent is that we will get it switched to be https, having https not be available locally would be a bad move.